README.md

```diff
@@ -5,6 +5,12 @@ Example automations for using Sessy with Home Assistant
 Controls Sessy to maintain value X on the grid meter.
 - Load balancing across multiple Sessy's
 - Except certain load via offset entities (e.g. EV charger)
+- Some attention points:
+  - Make sure you don't have any helper entities with `_power_strategy`, `_power_setpoint`, `_state_of_charge` or `_system_state` in the name
+  - `Update Interval` needs to be slower than the grid connection sensor update interval.
+  - A reasonable `timeout` value is 5 seconds
+  - `Smoothing` default of 100 is a good starting point. This is the maximum step adjustment. When your sensor updates more frequent you should lower the smoothing value. A large value may result in instable control 
+  - Your net grid consumption sensor needs to have the unit Watt
```

Comment:
This isn't strictly neccesary; if the unit is in kW, it will automatically be converted to W by the automation.